src/github.com/getlantern/autoupdate-server/server/asset_test.go

```diff
@@ -0,0 +1,16 @@
+package server
+
+import (
+	"fmt"
+	"testing"
+)
+
+const (
+	testAssetURL = `https://github.com/getlantern/autoupdate-server/releases/download/0.4.0/autoupdate-binary-darwin-amd64.v4`
+)
+
+func TestDownloadAsset(t *testing.T) {
+	if _, err := downloadAsset(testAssetURL); err != nil {
+		t.Fatal(fmt.Errorf("Failed to download asset: %q", err))
+	}
+}
```

Comment:
It might make more sense to make testpackages.txt a blacklist instead of a whitelist -- i.e. don't test these packages that aren't our code. Either that or just test everything in org/getlantern?